epsagon/modules/tornado.py

```diff
@@ -80,6 +91,11 @@ def after_request(cls, wrapped, instance, args, kwargs):
 
             tornado_runner = cls.RUNNERS.pop(unique_id)
 
+            # Ignoring 404s
+            if getattr(instance, '_status_code', None) == 404:
+                print_debug('Ignoring 404 Tornado request')
```

Comment:
should pop the trace from trace_factory (and prevent memory leaks for those 404 traces)